app/controllers/deploys_controller.rb

```diff
@@ -60,7 +60,7 @@ def recent
   #   * status (what is the status of this job failed|running| etc)
 
   def search
-    if (params[:status] && !Job.valid_status?(params[:status]))
+    if (params[:status].present? && !Job.valid_status?(params[:status]))
```

Comment:
updated code -> updated test ?

could use `status = status.presence` to not repeat the `.present?`check